src/resolvers/workspace.js

```diff
@@ -320,14 +320,66 @@ module.exports = {
         );
 
         if (!isThereOtherAdmins) {
-          throw new ForbiddenError('You can\'t leave this workspace because you are the last admin');
+          return false;
         }
       }
       await workspaceModel.removeMember(userModel);
 
       return true;
     },
 
+    /**
+     * Mutation in order to leave workspace
+     * @param {ResolverObj} _obj - object that contains the result returned from the resolver on the parent field
+     * @param {string} workspaceId - id of the workspace where the user should be removed
+     * @param {UserInContext} user - current authorized user {@see ../index.js}
+     * @param {ContextFactories} factories - factories for working with models
+     * @return {Promise<boolean>} - true if operation is successful
+     */
+    async deleteWorkspace(_obj, { workspaceId }, { user, factories }) {
+      const workspaceModel = await factories.workspacesFactory.findById(workspaceId);
+
+      if (!workspaceModel) {
+        throw new UserInputError('There is no workspace with provided id');
+      }
+
+      const memberInfo = await workspaceModel.getMemberInfo(user.id);
+
+      /**
+       * Return if user is not Admin.
+       */
+      if (!memberInfo.isAdmin) {
+        return false;
+      }
+
+      const membersInfo = (await workspaceModel.getMembers());
+
+      for (const member of membersInfo) {
+        const userModel = await factories.usersFactory.findById(member.userId.toString());
```

Comment:
> member.userId.toString()

there will be an error if the user is just invited by email, but has not accepted the invitation

<img width="688" alt="image" src="https://user-images.githubusercontent.com/3684889/151407012-11807602-4131-40a4-b459-2dd1c06c658d.png">

<img width="697" alt="image" src="https://user-images.githubusercontent.com/3684889/151407171-79f8647c-c684-44d1-b40c-f4124ddb5979.png">


-----


The second question: what will happen when we will invite the user, then remove the workspace, and then the user will open the invitation link?